spec/jstests-spec.js

```diff
@@ -290,12 +294,36 @@ describe("Pressing next and previous buttons associated with a graph", function
     expect(currMutationIndex).toBe(-1);
     expect(nextButton.disabled).toBe(false);
     expect(prevButton.disabled).toBe(true);
+  });
+  it("Navigate graph returns with no mutations doesn't change anything", function() {
+    document.body.appendChild(numDisplay);
+    initializeNumMutations(-1);
+    setCurrMutationNum(10);
+   
+   // Nothing changes with a negative numMutations
+    navigateGraph(-1);
+    expect(currMutationNum).toBe(10);
 
-    nextButton.click();
-    expect(currMutationNum).toBe(0);
+    navigateGraph(1);
+    expect(currMutationNum).toBe(10);
+  });
+  it("Index is a decimal navigate forward", function() {
```

Comment:
nit: the description for the `it` function is appended to the description of `describe` "Pressing next and previous buttons associated with a graph" when tests are run, so its generally a convention and helps readability to write the description for each test such that it connects to a full sentence.
e.g. `it("correctly navigates forward when index is a decimal")` => "Pressing next and previous buttons associated with a graph correctly navigates forward when index is a decimal"